pkg/apis/machine/v1beta1/v1beta1_suite_test.go

```diff
@@ -51,13 +53,22 @@ var (
 )
 
 func TestMain(m *testing.M) {
+	validatingHooks := NewValidatingWebhookConfiguration()
```

Comment:
validating**Hooks** vs mutating**Webhooks** vs Validating**WebhookConfiguration**
Can we please be consistent with naming?